test/WebJobs.Script.Tests/CSharpEndToEndTests.cs

```diff
@@ -32,6 +32,25 @@ public CSharpEndToEndTests(TestFixture fixture) : base(fixture)
         }
 
         [Fact]
+        public async Task NotificationHub()
+        {
+            await NotificationHubTest();
+        }
+
+        [Fact]
+        public async Task NotificationHub_Out_Notification()
+        {
```

Comment:
I talked with @fabiocav while I was writing this. It's by-design that nuget does not do a restore on startup and instead only reacts when project.json is changed. If I remember the conversation correctly -- in practice, the host will always be running when a project.json file appears or is changed. So our e2e tests need to simulate that.